src/main/java/com/ryanpmartz/booktrackr/config/ApiDocumentationConfig.java

```diff
@@ -31,4 +33,13 @@ public Docket apiDocketConfiguration() {
                 .paths(PathSelectors.any())
                 .build();
     }
+
+    @Bean
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/martzrp/booktrackr/file/2580809676/issues/source?bid=3283479&fileBranchId=3302859#l37)